lib/net/modules/dns.toit

```diff
@@ -39,7 +60,7 @@ If there are multiple servers then they are tried in rotation until one
   the next one.  This is in line with the way that Linux handles multiple
   servers on the same lookup request.
 */
-dns-lookup -> net.IpAddress
+dns-lookup-ips -> List
```

Comment:
Maybe `-all` or `-multi` instead of `-ips`?

Reply:
I like -multi